script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -335,175 +406,196 @@ His screams reverberate throughout the entire room.
 [sha:ce267a630912217568131ae4419f0cad40c3663f]{
 -- Page 449, Offset 19385.
 -- その<光|・><景|・>を、残った５人の若者たちは震えながら眺める事しかできなかった。
-Trembling, the remaining five young men and women could do nothing but watch the show unfold before their very eyes.
+The five vampires that remained could do nothing but cower and watch the %{i}show%{/i} unfolding before them.
 }
 [sha:5f0aa9df7a4901665a08bbae4ebc6d337fc90df8]{
 -- Page 449, Offset 19386.
 -- シスターは槍から手を放すと、丹念に、執拗に、倒れた獲物に処置を施した。
-The sister releases her grip on the spear, then painstakingly and tenaciously executes her collapsed prey.
+The nun releases her grip on the spear, and begins to meticulously and ruthlessly execute her downed prey.
 }
 [sha:9d508974dac8f0a816c9dbcc019853adf2c77f9f]{
 -- Page 450, Offset 19387. Mods: @n.
 -- まずは秘蹟によって通常の痛覚を復活させる。目を閉じられないように目蓋を釘で固定する。
-First, restoring a normal sense of pain with a Sacrament. Next, fastening his eyelids down with nails so they remain open.
+First, she uses a sacrament to restore their sense of pain.
+ That done, she pins their eyelids in place with nails so that they cannot look away.
 }
 [sha:bef5ef3d254058bfc88b9ea6c524e9b21eb850cb]{
 -- Page 450, Offset 19388. Glued.
 -- 服を脱がして裸身にすると皮を剥いで中身をあらわにする。
-Taking his clothes off, rendering him naked and removing his skin, exposing his innnards.
+ She then removes his clothes, and once he's fully naked, starts to peel back his skin, exposing his innards.
 }
 [sha:6418495d956c7d4948230c01affa3de91d89d342]{
 -- Page 451, Offset 19389. Mods: @n.
 -- 内臓の一つ一つを黒鍵で抉り取る。
-Gouging out his internal organs one by one.
+// This page is pretty tight. At least it's all glued so we don't lose as much to line accounting.
+One by one, she gouges out his organs with a black key.
 }
 [sha:56dd665bbd801de43e7ab9de30eb48ff8c4ddeb1]{
 -- Page 451, Offset 19390. Glued. Mods: @n.
 -- もちろんそのたびに死に至る激痛が獲物を襲う。内臓をすべて消費した後は耳と鼻をそぎ落とす。変わり果てた自分の姿を見て獲物がモウ殺シテと嘆願する。
-Of course, he's assaulted by a fatal pain every time. When his internal organs have all been gouged out, next comes the shearing of his ears and nose. Seeing his mangled form, he will beg to be killed.
+ Of course, each strike inflicts a lethal dose of pain.
+ Once she's done with his organs, she rips off his nose and ears.
+ If her prey could see what he's become, he'd beg her to just kill him.
 }
 [sha:e3048536b20af97e936beeb8bab957c582d1d408]{
 -- Page 451, Offset 19391. Glued. Mods: @n.
 -- 最後に頭蓋をパックリと割る。大皿のようなその容器に聖水を流し込む。
-Finally, spitting the skull open and pouring holy water into the vessel.
+// Tfw no trepanning nun gf
+ Finally, she punctures a hole in his skull and fills it with holy water.
 }
 [sha:371ff6392a9cccc63a5c349a8d2bdf794e81fe7c]{
 -- Page 451, Offset 19392. Glued. Mods: @n.
 -- 彼等にとって聖水は硫酸に等しい。脳に痛覚はない筈だが、洗礼によって清められた液体は痛覚以上の痛みで獲物を包み込み、
-To them, holy water is like sulfuric acid. The brain shouldn't have a sense of pain, but the liquid purified by baptism envelops the prey with an unimaginable agony and―――
+ To them, holy water is like sulfuric acid.
```

Comment:
woah,  a reference to the hit Netflix original series Dahmer: A Monster: Jeffrey Dahmer: A Story